File: Day11/Challenge1.py
## Challenge

Reverse String - Multiples of 3
The program must accept a string S containing only alphabets as the input. The program must reverse the string S. Then the program must print the alphabets which are 
present at the positions of multiples of 3 in the modified string as the output.
Boundary Condition(s):
3 <= Length of S <= 1000
Input Format:
The first line contains S.
Output Format:
The first line contains the alphabets as per the given condition.
Example Input/Output 1:
Input:
embezzling
Output:
izm
Explanation:
After reversing the string embezzling, the string becomes gnilzzebme.
In the string gnilzzebme, the alphabets present at the positions of multiples of 3 are i, z and m.
Hence the output is izm
Example Input/Output 2:
Input:
Nature
Output:
uN

## Solution

s= input()
rev= s[::-1]
l= len(rev)
for i in range(l):
    if (i+1)%3==0:
        print(rev[i], end='')

File: Day35/Challenge1.java
//Challenge - Reverse Every Word

/*
The program must accept a string S containing multiple words as the input. The program must reverse
every word in the string S. Then the program must print the modified string S as the output.
Boundary Condition(s):
5 <= Length of S <= 1000
Input Format:
The first line contains S.
Output Format:
The first line contains the modified string S.
Example Input/Output 1:
Input:
Friday and Saturday
Output:
yadirF dna yadrutaS
Explanation:
After reversing every word in the string Friday and Saturday, the string becomes yadirF dna
yadrutaS.
Hence the output is yadirF dna yadrutaS
Example Input/Output 2:
Input:
Honesty is the best policy
Output:
ytsenoH si eht tseb ycilop
*/

//Solution

import java.util.*;
public class Hello {

    public static void main(String[] args) {
		Scanner sc= new Scanner(System.in);
		String s= sc.nextLine();
		String[] words= s.split(" ");
		StringBuilder revst= new StringBuilder();
		for(String word:words){
		    StringBuilder sb= new StringBuilder(word);
		    revst.append(sb.reverse().append(" "));
		}
		revst.toString().trim();
        System.out.print(revst);
	}
}
